### ISSUE TITLE
couchdb_os_daemons_tests fails assertEqual

### ISSUE THREAD

**Issue by wohali** [apache/couchdb#675]
## Expected Behavior
`couchdb_os_daemons_tests:187: should_die_quickly` should pass...

## Current Behavior
```
    couchdb_os_daemons_tests:187: should_die_quickly...*failed*
in function couchdb_os_daemons_tests:'-check_dead/2-fun-3-'/2 (test/couchdb_os_daemons_tests.erl, line 231)
in call from couchdb_os_daemons_tests:check_dead/2 (test/couchdb_os_daemons_tests.erl, line 231)
in call from couchdb_os_daemons_tests:should_halts/2 (test/couchdb_os_daemons_tests.erl, line 210)
**error:{assertEqual_failed,[{module,couchdb_os_daemons_tests},
                     {line,231},
                     {expression,"D # daemon . status"},
                     {expected,halted},
                     {value,running}]}
```

Jenkins run:
* https://builds.apache.org/blue/organizations/jenkins/CouchDB/detail/master/87/pipeline/49

**Comment by wohali** [comment]
Again @ https://builds.apache.org/blue/organizations/jenkins/CouchDB/detail/master/92/pipeline/49

**Comment by wohali** [comment]
Fixed by #679. Will reopen if we see this recur.
